style.css

```diff
@@ -0,0 +1,36 @@
+#main {
+  /* margin: 0 auto; */
+
+  /* text-align: center; */
```

Comment:
- [ ] Please remove commented lines.

Reply:
I think it would be good to give a reason why commented lines should be removed, as this would give more context to the change requested.